packages/core/src/context/ContextSystem.ts

```diff
@@ -285,6 +285,13 @@ export class ContextSystem extends System
     {
         const attributes = gl.getContextAttributes();
 
+        const isWebGl2 = gl instanceof window.WebGL2RenderingContext;
```

Comment:
This will throw an error if `window.WebGL2RenderingContext` is undefined. Which is part of the reason for having the `'WebGL2RenderingContext' in window` check.